model/src/main/java/com/epam/esm/entity/OrderItem.java

```diff
@@ -1,13 +1,20 @@
 package com.epam.esm.entity;
 
 import com.epam.esm.dto.CartItem;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import javax.persistence.*;
```

Comment:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad
Please customize your IDE.
https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij

Reply:
done